.circleci/config.yml

```diff
@@ -79,14 +79,11 @@ jobs:
     steps:
       - setup
       - run:
-          name: "make build-all"
-          command: make build-all
-      - run:
-          name: "make docker"
-          command: make docker
+          name: "install"
+          command: make install
       - run:
           name: "make test"
           command: |
-            mkdir -p $HOME/testground/plans
-            ln -s $HOME/project/plans/placebo $HOME/testground/plans/placebo
+            testground plan import --from $HOME/project/plans/placebo
+            testground plan import --from $HOME/project/plans/example
```

Comment:
nice

Reply:
I’ve create #1010 for Cory to update relevant docs

Reply:
Doc-ed!